core/unit_test/TestReducers.hpp

```diff
@@ -1036,10 +1028,6 @@ struct TestReducers {
   static void execute_bool() {
     test_LAnd(10001);
     test_LOr(35);
-    test_sum(10001);
-    test_prod(35);
-    test_min(10003);
-    test_max(10007);
```

Comment:
This should be the only change necessary in this file. We shouldn't delete the tests for other scalar types above.

Reply:
What Daniel said.